src/olympia/amo/tests/test_commands.py

```diff
@@ -115,6 +125,43 @@ def test_compress_assets_correctly_fetches_static_images(settings, tmpdir):
         assert 'img/icons/stars.png' in data
 
 
+@pytest.mark.static_assets
+def test_compress_assets_correctly_compresses_js(settings, tmpdir):
+    """
+    Make sure that `compress_assets` correctly calls uglifyjs and that it
+    generates a minified file.
+    """
+    settings.MINIFY_BUNDLES = {
+        'js': {'zamboni/_test_js': ['js/zamboni/global.js']}}
+
+    js_all = os.path.join(
+        settings.STATIC_ROOT, 'js', 'zamboni', '_test_js-all.js')
+    js_min = os.path.join(
+        settings.STATIC_ROOT, 'js', 'zamboni', '_test_js-min.js')
+
+    # Delete the files if they exist - they are specific to tests.
+    try:
+        os.remove(js_all)
+    except FileNotFoundError:
+        pass
+    try:
+        os.remove(js_min)
+    except FileNotFoundError:
+        pass
+
+    # Capture output to avoid it being logged and allow us to validate it
+    # later if needed
+    out = io.StringIO()
+
+    # Now run compress and collectstatic
+    call_command('compress_assets', force=True, stdout=out)
+    call_command('collectstatic', interactive=False, stdout=out)
```

Comment:
I assume we need to call both of these to get the files to be written?

Reply:
Technically, only the first one is needed to have files written, but `collecstatic` moves static files from all apps to the final location where they will be served from, so it's useful to have this here as well to test the full config/behavior.